tests/WatchTests.cs

```diff
@@ -91,19 +91,10 @@ public void SuriveBadLine()
                 httpContext.Response.ContentLength = null;
 
                 await WriteStreamLine(httpContext, MockKubeApiServer.MockPodResponse);
-                await Task.Delay(TimeSpan.FromMilliseconds(100));
```

Comment:
those delays are to simulate that server is emitting messages

Reply:
But why do we need delays? At the end of the day it's just a byte stream, right?